### PR TITLE
[Computer Vision] Add new read operation for OCR API.

### PR DESCRIPTION
**Pull request by toothache**
## Description
Add read operation for OCR API.

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**
     `RecognitionResult` is renamed to `TextRecognitionResult`.

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by toothache** [comment]
A PR for updating swagger spec changes has been submitted.

https://github.com/Azure/azure-rest-api-specs/pull/5314

File: src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Generated/SdkInfo_ComputerVisionClient.cs [hunk 1]
```diff
@@ -45,10 +48,10 @@ internal static partial class SdkInfo
       // BEGIN: Code Generation Metadata Section
       public static readonly String AutoRestVersion = "latest";
       public static readonly String AutoRestBootStrapperVersion = "autorest@2.0.4283";
-      public static readonly String AutoRestCmdExecuted = "cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/ComputerVision/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=C:\\Users\\mamirzam\\CV-API\\azure-sdk-for-net\\src\\SDKs";
-      public static readonly String GithubForkName = "Azure";
-      public static readonly String GithubBranchName = "master";
-      public static readonly String GithubCommidId = "a2323a5bf8cd642bee954f7b49ff4fe1976cdeec";
+      public static readonly String AutoRestCmdExecuted = "cmd.exe /c autorest.cmd https://github.com/toothache/azure-rest-api-specs/blob/ocr_read/specification/cognitiveservices/data-plane/ComputerVision/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=E:\\Source\\cognition\\azure-sdk-for-net\\src\\SDKs";
+      public static readonly String GithubForkName = "toothache";
```
Comment (toothache) [inline src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Generated/SdkInfo_ComputerVisionClient.cs]: Will generate SDK using code from master branch of `azure-rest-api-specs` once the [PR](https://github.com/Azure/azure-rest-api-specs/pull/5314) is approved and gets merged.

[Computer Vision] Add read to CV OCR API. #5314
https://github.com/Azure/azure-rest-api-specs/pull/5314

File: src/SDKs/_metadata/cognitiveservices_data-plane_ComputerVision.txt [hunk 2]
```diff
@@ -3,12 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/ComputerVision/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=C:\Users\mamirzam\CV-API\azure-sdk-for-net\src\SDKs
-2018-12-11 18:58:56 UTC
+cmd.exe /c autorest.cmd https://github.com/toothache/azure-rest-api-specs/blob/ocr_read/specification/cognitiveservices/data-plane/ComputerVision/readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=E:\Source\cognition\azure-sdk-for-net\src\SDKs
```
Comment (toothache) [inline src/SDKs/_metadata/cognitiveservices_data-plane_ComputerVision.txt]: Will generate SDK using code from master branch of azure-rest-api-specs once the [PR](https://github.com/Azure/azure-rest-api-specs/pull/5314) is approved and gets merged.

[Computer Vision] Add read to CV OCR API. #5314

**Review by Zhouyi011 — APPROVED** [review]

**Review by toughever — APPROVED** [review]

**Comment by dsgouda** [comment]
@toothache please pull changes from master and update the PR. Also please note that our default branch has been switched to master

File: src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Generated/Models/BatchReadFileHeaders.cs [hunk 3]
```diff
@@ -11,26 +11,29 @@
...
+    /// Defines headers for BatchReadFile operation.
+    /// </summary>
+    public partial class BatchReadFileHeaders
     {
         /// <summary>
-        /// Initializes a new instance of the RecognitionResult class.
+        /// Initializes a new instance of the BatchReadFileHeaders class.
         /// </summary>
-        public RecognitionResult()
+        public BatchReadFileHeaders()
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Generated/Models/BatchReadFileHeaders.cs]: This is a breaking change, please bump the Nuget version to `4.0.0` and update the csproj/AssemblyInfo.cs files

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Generated/Models/BatchReadFileHeaders.cs (hunk 3 above)
Comment (toothache) [reply src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Generated/Models/BatchReadFileHeaders.cs]: Thanks. Nuget version bumped to `4.0.0`.

Besides, I think GIT diff is showing a wrong comparison here. 

`Models/RecognitionResult.cs` → `Models/TextRecognitionResult.cs`
`Models/BatchReadFileHeaders.cs` is a newly introduced file.

**Comment by toothache** [comment]
> @toothache please pull changes from master and update the PR. Also please note that our default branch has been switched to master

Thanks. I have rebased to master and pull latest changes from spec.

**Comment by shahabhijeet** [comment]
@AlexGhiondea please review this PR and approve/disapprove.

File: src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Microsoft.Azure.CognitiveServices.Vision.ComputerVision.csproj [hunk 4]
```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Vision.ComputerVision</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services ComputerVision APIs.</Description>
-    <Version>3.4.0</Version>
+    <Version>4.0.0</Version>
     <AssemblyName>Microsoft.Azure.CognitiveServices.Vision.ComputerVision</AssemblyName>
     <PackageTags>Microsoft Cognitive Services;Cognitive Services;Cognitive Services SDK;REST HTTP client;Computer Vision;Computer Vision API;Computer Vision SDK;Vision;netcore451511</PackageTags>
     <PackageReleaseNotes>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Microsoft.Azure.CognitiveServices.Vision.ComputerVision.csproj]: Please update the `PackageReleaseNotes` here

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Microsoft.Azure.CognitiveServices.Vision.ComputerVision.csproj (hunk 4 above)
Comment (toothache) [reply src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Microsoft.Azure.CognitiveServices.Vision.ComputerVision.csproj]: Updated.

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
Will merge when CIs pass

**Pull request merged**